examples_tests/23.Autoexposure/fstFrag.frag

```diff
@@ -0,0 +1,19 @@
+#version 430 core
+
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+// vertex shader is provided by the fullScreenTriangle extension
+
+layout (set = 3, binding = 0) uniform usampler2DArray tex0;
+
+layout(location = 0) in vec2 TexCoord;
+
+layout(location = 0) out vec4 pixelColor;
+
+void main()
+{
+    const uint packed = texture(tex0, vec3(TexCoord.x, TexCoord.y, 1.0)).r;
+    pixelColor = unpackUnorm4x8(packed);
+}
```

Comment:
why are you using some custom shader to unpack an R32_UINT texture as an RGBA?

could you not have used the already present builtin FS-triangle shader + an  image view with a correct format?

Reply:
@nipunG314 we already have some HLSL shader for dispaying a simple image from a UV coord, don't write custom